demo/src/main/resources/templates/webshop.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>webshop</title>
+</head>
+<body>
+
+</body>
+</html>
```

Comment:
Why do we need this?

Reply:
still waiting for the answer

Reply:
@deb05121 no answer received

Reply:
@deb05121 The whole html page is unnecessary, since you are having `REST` end points and you are returning JSONs, not HTML pages.

Reply:
HTML pages are needed when you are using controllers, not rest controllers and you want to return dynamically created web pages

Reply:
@deb05121 please address this - you don't need this html at all